styles/mixins.less

```diff
@@ -42,7 +39,6 @@
     background: -webkit-gradient(radial, center center, 0px, center center, 100%, color-stop(0%, @startColor), color-stop(100%,  @endColor)); /* Chrome,Safari4+ */
     background: -webkit-radial-gradient(center, ellipse cover, @startColor 0%, @endColor 100%); /* Chrome10+, Safari5.1+ */
     background:    -moz-radial-gradient(center, ellipse cover, @startColor 0%, @endColor 100%); /* FF3.6+ */
-    background:     -ms-radial-gradient(center, ellipse cover, @startColor 0%, @endColor 100%); /* IE10+ */
```

Comment:
This is for IE 10+. What background will be in IE 11 ?

Reply:
radial-gradient without prefix works well for IE11. https://caniuse.com/#feat=css-gradients

Reply:
What about other removed "-ms" prefixes?

Reply:
Removed prefixes do not require for IE11. I have found documentation or wrote an example for every property